### ISSUE TITLE
Fails to launch

### ISSUE THREAD

**Issue by xscd** [flathub/org.telegram.desktop#312]
In the latest update Telegram fails to launch. I get: 
```
telegram-desktop: error while loading shared libraries: libdav1d.so.4: cannot open shared object file: No such file or directory

```

**Comment by gasinvein** [comment]
A fix 1cc075e2b385171ecc5cf1983df2b28d8c56849f is on its way. As a workaround, you can run Telegram with
`flatpak run --env=LD_LIBRARY_PATH=/usr/lib/x86_64-linux-gnu org.telegram.desktop`

**Comment by xscd** [comment]
The fix deployed. ```flatpak update``` resolves the issue, thanks!
